src/console.c

```diff
@@ -2317,7 +2317,8 @@ static void tryReloadCode(Console* console, char* codeBuffer)
 
 static void cmdInjectCode(Console* console, const char* param, const char* name)
 {
-	if(strcmp(param, "-code") == 0)
+	bool watch = strcmp(param, "-code-watch") == 0;
+	if(strcmp(param, "-code") == 0 || watch)
```

Comment:
Micro-optimization: format as `if(watch || strcmp(param, "-code") == 0)` to prevent doing 2 `strcmp`s when only 1 is needed.

Reply:
Fair enough!